README.md

```diff
@@ -5,6 +5,7 @@
 Node 4.x (and above) is **required**. We suggest using [nvm](https://github.com/creationix/nvm) or [n](https://github.com/tj/n) to keep multiple Node versions on your system.
 
 
+
```

Comment:
Could you maybe add some comments that explains this change? Nothing huge, but two or three paragraphs about what's going on should be sufficient!